app/Http/Requests/UpdateAssertionRequest.php

```
@@ -0,0 +1,42 @@
+
+        return $monitor instanceof Monitor && $this->user()?->can('update', $monitor);
+    }
+
+    /**
+     * @return array<string, ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'type' => ['sometimes', 'string', 'in:status,latency,body,header,content_type'],
+            'expression' => ['sometimes', 'string', 'max:512', function (string $attribute, mixed $value, \Closure $fail) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attribute'. (undefined)

(UnusedFormalParameter)